core/src/main/java/com/netflix/iceberg/ManifestReader.java

```diff
@@ -62,7 +62,18 @@
    * @return a manifest reader
    */
   public static ManifestReader read(InputFile file) {
-    return new ManifestReader(file);
+    return new ManifestReader(file, true);
+  }
+
+  /**
+   * Returns a new {@link ManifestReader} for an {@link InputFile}.
+   *
+   * @param file an InputFile
+   * @param caseSensitive whether column name matching should have case sensitivity
+   * @return a manifest reader
+   */
+  public static ManifestReader read(InputFile file, boolean caseSensitive) {
```

Comment:
I think manifests should set case sensitivity using a refinement method, like `TableScan.caseSensitive(boolean)`. I generally like to avoid boolean parameters in public methods because it isn't obvious what the boolean flag is without editor support.

Reply:
Will modify.

Reply:
This is no longer needed because the reader has a `caseSensitive` refinement method, right?